packages/rum/src/domain/record/observers.spec.ts

```diff
@@ -85,6 +95,13 @@ describe('initInputObserver', () => {
     input.value = newValue
     input.dispatchEvent(createNewEvent('input', { target: input }))
   }
+
+  function dispatchComposedInputEvent(newValue: string) {
+    input.value = newValue
+    const event = createNewEvent('input', { target: sandbox, composed: true })
+    spyOn(event, 'composedPath').and.returnValue([input, sandbox, document.body])
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏no need for a Spy here:

```js
event.composedPath = () => [...]
```